source/Ubuntu-R.ufo/fontinfo.plist

```diff
@@ -107,9 +107,9 @@ The scope of the Ubuntu Font Family includes all the languages used by the vario
 		<array>
 		</array>
 		<key>openTypeNameUniqueID</key>
-		<string>Ubuntu Regular Version 0.83</string>
+		<string>Ubuntu Regular Version 1.000</string>
 		<key>openTypeNameVersion</key>
-		<string>Version 0.83</string>
+		<string>Version 1.000</string>
```

Comment:
The addition of the *code* to auto-update the version number should be completely separate from the proposal to bump the version number to `1.0`.  For the moment it should still generate `0.83`.

(Switching to a higher number should be the *absolutely very last change before making a release*.  This has caused so many problems in the past.